tests/unit/conftest.py

```diff
@@ -10,3 +10,32 @@ def root_dir(request):
 @pytest.fixture(scope="module")
 def local_env_file(root_dir):
     yield root_dir / "tests" / "test.env"
+
+
+@pytest.fixture(scope="session")
+def session_plex():
+    """
+    Fixture to return the prometheus log exporter module.
+    """
+    import sys
+    from pathlib import Path
+    import importlib.util
+
+    # Ensure the parent directory is in the sys.path
+    root = Path(__file__).parent.parent.parent
+    path = root  / "birdhouse" / "optional-components" / "prometheus-log-parser" / "config" / "thredds"
+    spec = importlib.util.spec_from_file_location("prometheus_log_exporter", str(path / "prometheus-log-exporter.py"))
+    plex = importlib.util.module_from_spec(spec)
+    sys.modules["prometheus_log_exporter"] = plex
+    spec.loader.exec_module(plex)
+    return plex
+
+
+@pytest.fixture(scope="function")
+def plex(session_plex):
+    """
+    Fixture to return a fresh instance of the prometheus log exporter module for each test.
```

Comment:
This isn't quite right. It doesn't give a fresh instance of the module, it just clears the previous stats from the `counter` object defined in the module. Can we reword this to:

```
Returns the prometheus log exporter module with an empty counter instance.
```

Reply:
yes